src/groups/unitary.jl

```diff
@@ -0,0 +1,220 @@
+@doc raw"""
+    Unitary{n,𝔽} <: AbstractGroupManifold{𝔽,MultiplicationOperation,DefaultEmbeddingType}
+
+The group of unitary matrices ``\mathrm{U}(n, 𝔽)``.
+
+The group consists of all points ``p ∈ 𝔽^{n × n}`` where
+``p^\mathrm{H}p = pp^\mathrm{H} = I``. All such points satisfy the property ``|\det(p)|=1``.
+
+The tangent vectors ``X_p ∈ T_p \mathrm{U}(n, 𝔽)`` are represented instead as the
+corresponding element ``X_e = p^\mathrm{H} X_p`` of the Lie algebra ``𝔲(n, 𝔽)``, which
+consists of the skew-hermitian matrices, that is, all ``X_e ∈ 𝔽^{n × n}`` where
+``X_e = -X_e^\mathrm{H}``.
+"""
+struct Unitary{n,𝔽} <: AbstractGroupManifold{𝔽,MultiplicationOperation,DefaultEmbeddingType} end
+
+Unitary(n, 𝔽::AbstractNumbers=ℝ) = Unitary{n,𝔽}()
```

Comment:
In most cases, I think people will use the term `Orthogonal` to refer to real unitary and `Unitary` with no qualifier to refer to complex unitary, so perhaps it makes sense to use this default:
```suggestion
Unitary(n, 𝔽::AbstractNumbers=ℂ) = Unitary{n,𝔽}()
```